warehouse/snowflake/snowflake_test.go

```diff
@@ -47,121 +37,109 @@ func (t *TestHandle) VerifyConnection() error {
 		return err
 	}
 
-	return testhelper.WithConstantBackoff(func() (err error) {
```

Comment:
I understand that verify the connection is used to wait for the snowflake to be available, thus is not needed to test both `db_name`s .

Reply:
Yes. Thant's true.